codecov.yml

```diff
@@ -1,2 +1,12 @@
 github_checks:
-  annotations: false
\ No newline at end of file
+  annotations: false
+
+coverage:
+  status:
+    project:
+      default:
+        target: auto
+        threshold: 2%  # allow for 2% reduction of coverage without failing
```

Comment:
I'm happy to discuss this. I have noticed it's not rare for the coverage to decrease a bit during PRs, and we ignore it most of the time